app/operations/story_operations.rb

```diff
@@ -49,6 +49,14 @@ def create_changesets
     end
   end
 
+  class UpdateAll < BaseOperations::UpdateAll
+    def operate!
+      model.map do |record|
```

Comment:
It seems like a duplicate of https://github.com/Codeminer42/cm42-central/pull/336/files#diff-e2654f79e5b86645e150ee633c098aefR74, is there something wrong there?